css/style.css

```diff
@@ -0,0 +1,143 @@
+body {
+  max-width: 100%;
+  padding: 1%;
+}
+
+.clearfix:after {
+  visibility: hidden;
+  display: block;
+  font-size: 0;
+  content: " ";
+  clear: both;
+  height: 0;
+}
+
+.hide {
+  display: none;
+}
+
+ul {
+  list-style: none;
+  padding-left: 4%;
+}
+
+h1 {
+  font-size: 1.5em;
+  font-weight: bold;
+  margin-top: 3%;
+}
+
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+header {
+  float: left;
+  padding-bottom: 2.25%;
+  width: 100%;
+}
+
+.arrow {
+  padding-left: 4%;
+  width: 6%;
+}
+
+.listorder--float {
+  float: left;
+}
+
+.logo {
+  float: left;
+  width: 20%;
+}
+
+.main__nav {
+  float: left;
+  width: 70%;
+}
+
+.main__nav li {
+  font-size: 1em;
+  font-weight: bold;
+  width: 25%;
+}
+
+.socialmedia {
+  float: left;
+  text-align: right;
+  width: 10%;
+}
+
+.socialmedia li {
+  width: 33.333%;
+}
+
+#secondary__nav--design--container, #secondary__nav--photography--container{
+  background: blue;
+  clear: both;
+  padding-top: 1%;
+  padding-bottom: 2%;
```

Comment:
Is the difference between 2% and 1% really visible and significant? Might be easier to maintain if you consolidated these to `padding: 1% 0` or something.